department-of-veterans-affairs/abd-vro
# Exclude RRD Shared

## What was the problem?
While trying to exclude RRD code in a previous PR, there was quite a bit of BIP code within rrd-shared which was a dependency of several other Gradle subprojects. 

Now that there is a directory location which will host the BIP-related code going forward, this PR moves related code from `rrd-shared` into that directory and excludes `rrd-shared` from build source.
Associated tickets or Slack threads:

- https://github.com/department-of-veterans-affairs/abd-vro/issues/1509

## How to test this PR
- Successful build and unit testing 
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
